manifest.json

```diff
@@ -23,7 +23,7 @@
     },
 
     "content_scripts": [{
-        "matches": ["VALID_URL_HERE", "VALID_URL_HERE"],
+        "matches": ["https://logs.devcerner.com/*search?*", "https://logs.devcerner.com/*search?*"],
```

Comment:
should this be reverted @InViN ?

Reply:
Yeah, just keep it as VALID_URL_HERE, thanks.